deploy_flask.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+TARGET="hello.py"
+
+apt update
+
+apt install -y python3.7 
+apt-get install -y python-pip 
+
+pip install --upgrade pip 
+pip install Flask 
+
+#go to the directory containing the file we want to run
+cd ..; cd ..; cd vagrant
+
+FLASK_APP=$TARGET flask run --host=0.0.0.0
```

Comment:
This is the only place TARGET is used.  Why not use the explicit name?

Reply:
easier access for future use, instead of diving into the code and looking for it (in a while I may forget where I placed it) just in case I want to use another file instead.